### ISSUE TITLE
Negative Loss

### ISSUE THREAD

**Issue by Jessika97** [NVlabs/geomapnet#39]
Dear Mr. Samarth Brahmbhatt,

I run your code on my own data and it turned out that I get negative values for the loss.
Is that intended? It just happens when I set beta and gamma to a negative value (default is -3) and activate the learning of both.
I also do not understand why beta and gamma are explained as translation and rotation balance weights in the paper, but in the code they are "saq" and "srq", which mean the absolute rotation weight and the relative rotation weight. 
Beta and gamma seem to get more negative without a lower bound in the training which also makes the loss more negative.

Thanks in advance.

**Comment by samarth-robo** [comment]
Hi @Jessika97 , 1) negative loss values are expected because of the log terms in the loss function, and 2) sorry for the mismatch in variable names. This is documented in a [README](https://github.com/NVlabs/geomapnet/blob/master/scripts/configs/README.md).
